R/UK.R

```diff
@@ -67,22 +67,33 @@ UK <- R6::R6Class("UK", # rename to country name
     download = function() {
       # set up filters
       self$set_filters()
-      message_verbose(self$verbose, "Downloading UK data.")
-      self$data$raw <- map(self$query_filters, self$download_uk)
-
+      if (self$verbose) {
+        message("Downloading UK data.")
+        self$data$raw <- map(self$query_filters, self$download_uk)
+      }
+      self$data$raw <- suppressMessages(
+        map(self$query_filters, self$download_uk)
+      )
       if (self$level == "1") {
         self$data$raw <- bind_rows(
           self$data$raw$nation, self$data$raw$region
         )
+        # get NHS data if requested
+        if (self$nhsregions) {
+          self$download_nhs_regions()
+        }
       } else if (self$level == "2") {
         self$data$raw <- self$data$raw[[1]]
+        self$download_authority_data()
```

Comment:
all looks. sensible. We really need some custom tests of this (using the custom testing framework implemented in #190.

Reply:
I agree, I will work on adding custom tests for these once it looks optimal